k8s/nginx-http/nginx-http-dep.yaml

```diff
@@ -12,7 +12,7 @@ spec:
       terminationGracePeriodSeconds: 10
       containers:
       - name: nginx
-        image: bigchaindb/nginx_http:1.1
+        image: bigchaindb/nginx_http:unstable
```

Comment:
See my other comments about the `unstable` images, and whether to change the `imagePullPolicy` or to adopt a naming convention such as `unstable1`, `unstable2`, etc.